src/trio/_tests/test_ssl.py

```diff
@@ -707,7 +707,7 @@ async def expect(expected: bytes) -> None:
     # Our receive_some() call will get stuck when it hits send_all
     async def sleeper_with_slow_send_all(method: str) -> None:
         if method == "send_all":
-            await trio.sleep(100000)
+            await trio.sleep(100000)  # noqa: ASYNC116  # not sleep forever
```

Comment:
Similar as for the other explanations for `ASYNC116`, could this explain the why rather than the what.

Reply:
Fixed in 924b6dd